scripts/check_frontend_coverage.py

```diff
@@ -537,35 +590,38 @@ def check_coverage_changes():
                         ' file does not exist.'.format(LCOV_FILE_PATH))
 
     stanzas = get_stanzas_from_lcov_file()
-    whitelist = list(FULLY_COVERED_FILENAMES)
+    blacklist = list(NOT_FULLY_COVERED_FILENAMES)
     errors = ''
 
     for stanza in stanzas:
         file_name = stanza.file_name
         total_lines = stanza.total_lines
         covered_lines = stanza.covered_lines
 
-        if file_name in whitelist:
+        if file_name not in blacklist:
             if total_lines != covered_lines:
-                errors += ('\033[1m{}\033[0m file is in the whitelist but its'
-                           ' coverage decreased. Make sure it is fully covered'
-                           ' by Karma unit tests.\n'.format(file_name))
-
-            whitelist.remove(file_name)
+                errors += ('\033[1m{}\033[0m seems to be not completely tested.'
+                           ' Make sure it\'s fully covered before pushing'
+                           ' your changes.\n'.format(file_name))
         else:
             if total_lines == covered_lines:
-                errors += ('\033[1m{}\033[0m file is fully covered but it\'s'
-                           ' not in the "100% coverage" whitelist. Please add'
-                           ' the file name in the whitelist in the file'
-                           ' scripts/check_frontend_test_coverage.py.\n'
+                errors += ('\033[1m{}\033[0m seems to be fully covered!'
+                           ' Before removing it manually from the blacklist'
```

Comment:
This should not be an error right? (To me this seems like a piece of advice)

Can you please explain when a user will see this error message? [Maybe I have misunderstood the situation]

Reply:
Ah, I realized this is for files is in the blacklist with 100% coverage, In that should we inform that they have to remove the filename from blacklist. The current instruction isn't clear to me! :(

Reply:
Can you please tell me where the instruction is not clear?

Reply:
I think we should have a clear instruction asking contributor to remove the file name from the `check_frontend_coverage.py`, currently, it starts with "Before" and I'm not sure what task I've to perform to resolve the issue.